crates/matrix-sdk-crypto/Cargo.toml

```diff
@@ -26,8 +26,7 @@ _disable-minimum-rotation-period-ms = []
 # "message-ids" feature doesn't do anything and is deprecated.
 message-ids = []
 
-# Testing helpers for implementations based upon this
-testing = ["dep:http"]
+testing = []
```

Comment:
since `matrix_sdk_crypto::machine::testing::response_from_file` no longer exists, we no longer need `http`.